superset-frontend/src/dashboard/components/dnd/DragDroppable.jsx

```diff
@@ -37,7 +37,7 @@ const propTypes = {
   component: componentShape.isRequired,
   parentComponent: componentShape,
   depth: PropTypes.number.isRequired,
-  disableDragDrop: PropTypes.bool,
+  disableDragdrop: PropTypes.bool,
```

Comment:
Nice catch 👍  I personally prefer `disableDragDrop`, but if it's used in many places it's probably better to just change it here.